### PR TITLE
Updated i18n-tool version, .travis.yml and Makefile files

### PR DESCRIPTION
**Pull request by iivic**
@clintonb Review changes.

### CODE REVIEW

**Comment by iivic** [comment]
Please review latest changes. @clintonb

File: Makefile [hunk 1]
```diff
@@ -46,7 +48,7 @@ requirements.js:
 	npm install
 	$(NODE_BIN)/bower install
 
-requirements:
+requirements: requirements.js
```
Comment (clintonb) [inline Makefile]: Revert. There may be workflows that depend on this.

File: .travis.yml [hunk 2]
```diff
@@ -24,7 +24,6 @@ before_install:
 install:
     - pip install -U pip wheel codecov
     - make requirements
-    - make requirements.js
```
Comment (clintonb) [inline .travis.yml:27]: Revert.

File: .travis.yml [hunk 3]
```diff
@@ -35,9 +34,11 @@ script:
     - cd docs && make html
     - cd ..
 
+    # Check if translation files are up-to-date
+    - make detect_changed_source_translations
```
Comment (clintonb) [inline .travis.yml]: `make validate_translations`

**Review by clintonb — CHANGES_REQUESTED** [review]

**Comment by iivic** [comment]
@clintonb I can't figure out why `make validate_translations` detect changes. Locally the source files are current. Can you take a look?

File: .travis.yml [hunk 4]
```diff
@@ -35,9 +35,11 @@ script:
     - cd docs && make html
     - cd ..
 
+    # Check if translation files are up-to-date
+    - make validate_translations
+
     # Compile static assets and validate the code
     - make static -e DJANGO_SETTINGS_MODULE="course_discovery.settings.test"
     - make validate
-
```
Comment (clintonb) [inline .travis.yml:41]: Revert.

**Review by clintonb — APPROVED** [review]
My fixup should resolve the build issue. Remember to squash before merging.

**Comment by iivic** [comment]
Thank you for the help. So the problem were the files that shouldn't be extracted?

**Comment by clintonb** [comment]
@iivic yes. We don't normally translate our documentation; although, we probably should consider doing so at some point.

**Pull request merged**